src/Pages/HomePage/HomePage.tsx

```diff
@@ -1,13 +1,14 @@
 
 import { NetComponent, Showcase, InfiniteSlider } from '../../Components/index';
 
+
 const HomePage = () => {
   return (
-    <>
+    <div style = {{overflow:'hidden'}}>
```

Comment:
inline styles  are not good practice in general. Better to use classnames

Reply:
Changed.